### ISSUE TITLE
block_store tests fail with debug build

### ISSUE THREAD

**Issue by conr2d** [NoirHQ/smite#113]
```
The following tests FAILED:
	 71 - block_store: save/load_block (Failed)
	 72 - block_store: prune_block (Failed)
Errors while running CTest
```

**Comment by xedonman** [comment]
This issue is fixed in #126, but still tests for macos fail in debug/release mode.
I will keep watching these failures in this issue

**Comment by xedonman** [comment]
blocking issue will be fixed by #162 
a "few sometimes" that hard to reproduce can be occur during automated test
those will be handled in new issue if exist

**Comment by xedonman** [comment]
#162 is merged to main.
thus close this issue
